nautobot/circuits/tests/test_models.py

```diff
@@ -17,16 +17,15 @@ def setUpTestData(cls):
         provider = Provider.objects.first()
         circuit_type = CircuitType.objects.first()
 
-        location_type_1 = LocationType.objects.get(name="Campus")
-        location_type_1.content_types.set([])
+        location_type_1 = LocationType.objects.create(name="University")
         location_type_2 = LocationType.objects.get(name="Building")
         location_type_2.content_types.add(ContentType.objects.get_for_model(CircuitTermination))
         status = Status.objects.get_for_model(Circuit).first()
         cls.circuit = Circuit.objects.create(
             cid="Circuit 1", provider=provider, circuit_type=circuit_type, status=status
         )
         cls.provider_network = ProviderNetwork.objects.create(name="Provider Network 1", provider=provider)
-        cls.location_1 = Location.objects.filter(location_type=location_type_1)[0]
+        cls.location_1 = Location.objects.create(name="Department", location_type=location_type_1, status=status)
```

Comment:
Shouldn't assume that `status` (retrieved above by `Status.objects.get_for_model(Circuit).first()`) is applicable to Locations as well - probably need to do a second status lookup here instead.